src/main/java/org/folio/inventory/validation/experimental/AbstractTargetValidator.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+import org.folio.inventory.support.http.server.ValidationError;
+
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
+
+public abstract class AbstractTargetValidator {
```

Comment:
As this validator is focused on the item status I think it would help comprehension for this to be included in the name

```suggestion
public abstract class AbstractTargetItemStatusValidator {
```